.github/workflows/validate-release.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
     env:
       CROSS_BUILDER_IMAGE: ghcr.io/gythialy/golang-cross:v1.18.5-1@sha256:7dda2158ee68f0e4f53ca9bdf3971b2db01084dad4bdc775391e5f3bf40056a5
```

Comment:
same version here, 1.19 ?

Reply:
yes, building the image right now :) will update soon
https://github.com/gythialy/golang-cross/runs/8108739745?check_suite_focus=true